src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,130 @@
+.main {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 1024px;
+  margin: auto;
+}
```

Comment:
these styles is not related to card
move to separate file or main.scss
for fix issue on screen do not set justify content. by default it will be start
![Screenshot from 2023-04-01 16-31-14](https://user-images.githubusercontent.com/33029864/229292325-bfe36d48-2f0a-44fc-b4a5-d17f51e736b2.png)